### ISSUE TITLE
A11y : Cannot reach tabs inside Inspector / Query History via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46477]
**Steps to reproduce**:

Tabs inside Inspector / Query History are not reachable by keyboard

**Actual Result**:

Basic functionality is lost for users who rely on keyboard navigation

**Expected Result**

Tabs should be accessible via keyboard navigation

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by L-M-K-B** [comment]
ToDo:
- [ ] make tab accessible
- [ ] make slider for time range accessible (query history tab)
- [ ] make tab content accessible

**Comment by L-M-K-B** [comment]
ToDo:
- [ ] make tab accessible
- [ ] make slider for time range accessible (query history tab)
- [ ] make tab content accessible
